firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/internal/ReleaseIdentificationUtils.java

```diff
@@ -62,6 +62,8 @@ public static String calculateApkInternalCodeHash(@NonNull File file) {
     try {
       MessageDigest digest = MessageDigest.getInstance("SHA-256");
       ArrayList<Byte> checksums = new ArrayList<>();
+      // hash the ZipFile to reuse existing checksum data for an intermediate hash which gets mapped
+      // to the codeHash returned from the release backend
```

Comment:
Can we update this comment to include the why behind using the zip? (In other words, can we answer the question of why we don't just calculate the codeHash returned from the backend?)